tasks/README.md

```diff
@@ -249,8 +248,8 @@ the scheduling for free, and is really easy to set up.
    up to 10 seconds and can fail, and HTTP requests ought to be replied fast
    and reliably.
 
- * Then all real test/release/etc. worker bots also connect to the same AMQP
-   container (locally through the service or remotely through the route).
+ * Then all worker bots also connect to the same AMQP container (locally
+ * through the service or remotely through the route).
```

Comment:
This bullet seems odd here

Reply:
Thanks! Overzealous auto-formatting in vim. Fixed.